Jenkinsfile

```
@@ -105,6 +105,15 @@ def runRelease(architectures) {
         version = getCliVersion(builderPath)
         print "CLI version: $version"
     }
+    /**
```

1. Add missing line break
2. Convert from JavaDoc style comment which meant to be above functions to a regular comment
```suggestion
    }

    /*
```